ush/regrid_gsiSfcIncr_to_tile.sh

```diff
@@ -79,63 +64,213 @@ else # construct restart times for deterministic member
     fi
 fi
 
-for imem in $(seq 1 "${NMEM_REGRID}"); do
-    if (( NMEM_REGRID > 1 )); then
-        cmem=$(printf %03i "${imem}")
-        memchar="mem${cmem}"
+#
+# Stage input files
+#
 
-        MEMDIR=${memchar} YMD=${PDY} HH=${cyc} declare_from_tmpl \
-            COMOUT_ATMOS_ANALYSIS_MEM:COM_ATMOS_ANALYSIS_TMPL
+# Create MDMD command file for fixed files
+rm -f cmdfile.0
+touch cmdfile.0
+chmod 755 cmdfile.0
+
+# Append fixed files command file to master command file
+{
+echo "#!/bin/bash" > cmdfile.0
```

Comment:
No need to pipe this out here since it's piped at the end of the block.
```suggestion
echo "#!/bin/bash"
```